src/views/app/TheIntegrations/components/PagoPlux.vue

```diff
@@ -0,0 +1,194 @@
+  secretKey: "",
+  active: false,
+  formVisible: false,
+});
+
+const toggleText = computed(() => {
+  return pagopluxForm.active ? "Desactivar" : "Activar";
+});
+
+const rules = {
+  ruc: rucRules,
+  clientId: pagopluxClientIdRules,
+  secretKey: pagopluxSecretKeyRules
+};
+
+const isFormValid = computed(() => {
+  const isRucValid = rucRules.every(rule => rule.validate(pagopluxForm.ruc));
+  const isClientIdValid = pagopluxClientIdRules.every(rule => rule.validate(pagopluxForm.clientId));
+  const isSecretKeyValid = pagopluxSecretKeyRules.every(rule => rule.validate(pagopluxForm.secretKey));
+  return isRucValid && isClientIdValid && isSecretKeyValid;
+});
+
+function isActive(event: boolean): void {
+  pagopluxForm.active = event;
+  if(!event) {
+    pagopluxForm.ruc = '';
+    pagopluxForm.clientId = '',
+    pagopluxForm.secretKey = ''
+  };
+  pagopluxForm.formVisible = event;
+};
+
+async function sendPagoPluxData () {
+  const data = {
+    restaurantRUC: pagopluxForm.ruc,
+    clientToken: `${pagopluxForm.clientId}:${pagopluxForm.secretKey}`
+  };
+  const restaurantUuid = clientStore.client?.restaurant?.uuid;
+  await paymentMethodsStore.putPagopluxData(data, restaurantUuid!);
+  pagopluxForm.formVisible = false;
+}
+
+onMounted( async () => {
+  const userAuth = await authStore.checkAuth();
+
+  await clientStore.getClientByEmail(userAuth?.email!);
+
+  await paymentMethodsStore.getPaymentMethods(clientStore.client?.restaurant?.uuid!)
+})
+</script>
+
+<template>
+  <p class="title">
+    <li>
+      Configura tu cuenta de pagoplux 😊
+    </li>
+  </p>
+  <Card :class="{ expanded: pagopluxForm.active }">
+    <template #title>
+      <div class="form-integration">
+        <p class="form-integration-title">Pagoplux</p>
+        <img :src="pagopluxImage" alt="logo de pagoplux" class="form-integration-image" />
+      </div>
+    </template>
+    <template #content>
+      <form class="form">
+        <ToggleInput
+          v-model:value="pagopluxForm.active"
+          :text="toggleText"
+          @update:modelValue="isActive" />
+        <div class="form-content" v-show="pagopluxForm.active && pagopluxForm.formVisible">
+          <CrushTextField
+            v-model="pagopluxForm.ruc"
+            :valid-rules="rules.ruc"
+            label="Ruc del establecimiento" 
+            placeholder="0954227382001" />
+          <CrushTextField
+            :valid-rules="rules.clientId"
+            v-model="pagopluxForm.clientId"
+            label="id del client" 
+            placeholder="k3rTOL7NCbGULs2m2jK6a04SUg"/>
+          <CrushTextField
+            :valid-rules="rules.secretKey"
+            v-model="pagopluxForm.secretKey"
+            label="Clave secreta" 
+            placeholder="pV5uMhA7oyQr0yXqB2bmoQkCtBfwivb5IVVz7pSUCCAWSDp3" />
+        </div>
+        <CrushButton
+          v-if="pagopluxForm.active && pagopluxForm.formVisible"
+          :disabled="!isFormValid"
+          @click.prevent="sendPagoPluxData"
```

Comment:
please do not use click.prevent, follow the way I use the click event, 
change the form for a div, in this case is valid because this should not affect our SEO because this is just for our users